src/script/lua_api/l_metadata.cpp

```diff
@@ -196,6 +196,42 @@ int MetaDataRef::l_set_float(lua_State *L)
 	return 0;
 }
 
+// get_bool(self, name)
+int MetaDataRef::l_get_bool(lua_State *L)
+{
+	MAP_LOCK_REQUIRED;
+
+	MetaDataRef *ref = checkAnyMetadata(L, 1);
+	std::string name = luaL_checkstring(L, 2);
+
+	IMetadata *meta = ref->getmeta(false);
+	if (meta == NULL) {
+		lua_pushboolean(L, false);
+		return 1;
+	}
+
+	std::string str_;
+	const std::string &str = meta->getString(name, &str_);
+	lua_pushboolean(L, str == "true" ? true : false);
```

Comment:
this should probably use `is_yes` see the implementation of `l_settings.cpp`

Reply:
Done (and removed the `? true : false` :sweat_smile:)